modules/ROOT/pages/jump-runtime-manager-task.adoc

```diff
@@ -3,15 +3,12 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-When you run an app in Design Center, it appears in Runtime Manager. You can jump to Runtime Manager from Options to manage the app there. In Runtime Manager, you can configure runtime operations of designs you run in Design Center.
+When you run an app in Flow Designer, it appears in the list of running applications in Runtime Manager. You can jump to Runtime Manager from Flow Designer to manage the app there. In Runtime Manager, you can configure runtime operations of Mule apps that you are designing.
 
-. Create an app and click Run.
-. In Design Center, click image:manage-connector-versions-design-center-ced9a.png[] Options.
-. Select View in Runtime Manager.
-+
-image::options-menu.png[]
+. If the status message in the toolbar above the canvas says, "Ready to Run", click *Run*.
+. In Design Center, click the three-dots icon to the left of the *Deploy* button.
```

Comment:
Is this Design Center supposed to be Flow Designer?

Reply:
The sentence makes more sense starting at "Click". I'll remove the phrase.